src/components/scenes/Fio/FioDomainSettingsScene.tsx

```diff
@@ -120,7 +120,7 @@ export class FioDomainSettingsComponent extends React.Component<Props, State> {
 
     const { currencyCode, pluginId } = fioWallet.currencyInfo
     onLogEvent('Fio_Domain_Renew', {
-      nativeAmount: renewalFee,
+      nativeAmount: String(renewalFee),
```

Comment:
Very bizarre that `renewalFee` was a number. Pls double check to make sure.

Reply:
Yes it was a number